bigbluebutton-html5/imports/api/2.0/users/server/modifiers/kickUser.js

```diff
@@ -0,0 +1,39 @@
+import { check } from 'meteor/check';
+import Logger from '/imports/startup/server/logger';
+import Users from './../../';
+
+export default function kickUser(meetingId, userId) {
+  check(meetingId, String);
+  check(userId, String);
+
+  const selector = {
+    meetingId,
+    userId,
+  };
+
+  const modifier = {
+    $set: {
+      'user.kicked': true,
```

Comment:
I don't see us resetting `kicked` to false in addUser. Please double check if we even need to set a property here

Reply:
double checked and user.kicked does need to be set to true.

Reply:
probably should be renamted to `kicked` since we dont have the nested user anymore.